javascript/code-challenges/linked-list/linkedList.js

```diff
@@ -90,6 +90,21 @@ class LinkedList {
       currentNode = currentNode.next;
     }
   }
+
```

Comment:
On the top line of requiring in linked list, it should say:

`const { LinkedList, Node } = require('../linked-list/linkedList');`